metpy/calc/tests/test_kinematics.py

```diff
@@ -363,3 +364,11 @@ def test_geostrophic_gempak():
                        [-8.66612, -5.27816, -1.45282]])
     assert_almost_equal(ug[1, 1], true_u[1, 1] * units('m/s'), 2)
     assert_almost_equal(vg[1, 1], true_v[1, 1] * units('m/s'), 2)
+
+
+def test_streamfunc():
+    """Test of Montgomery Streamfunction calculation."""
+    t = 287. * units.kelvin
+    hgt = 5000. * units.meter
+    msf = montgomery_streamfunction(hgt, t)
+    assert_almost_equal(msf, 337468.25 * units('m/s') * units('m/s'), 2)
```

Comment:
Can simplify to: `337468.25 * units('m^2 s^-2')`

I'd go ahead and test to at least 4 places after the decimal.